docs/src/content/quickstart.md

```diff
@@ -4,4 +4,35 @@ name: Quickstart
 
 # Quickstart Guide
 
-This will lead you to different paths.
+This Quickstart tutorial will guide you to setup the Hydrus server, we will be using docker in this tutorial,
+Therefore you can go through the [Docker container engine](https://docker-curriculum.com/) and its best practices deploying virtualized environments.
```

Comment:
Users don't need to know the best practices to follow this document, right?

Reply:
Yes, you are right my intention was to just make people familiar with docker many of them may not aware of docker-compose.